jobs/nightly-samba-rpm-builds.yml

```diff
@@ -12,30 +12,30 @@
         - origin/master
 
     properties:
-    #- github:
-    #    # TODO: change the repo to final location
-    #    url: https://github.com/obnoxxx/samba-integration
+    - github:
+        url: https://github.com/gluster/samba-integration
     - build-discarder:
         days-to-keep: 7
-        #artifact-days-to-keep: 7
+        artifact-days-to-keep: 7
```

Comment:
This does not matter much. It refers to the artifacts that Jenkins gathers. The rsync'd repository needs to be cleaned on the CentOS side.

Reply:
Sure, I just noticed that I had commented that out in the initial commit.
I'll leave that hunk alone for this commit.

Reply:
left the artifacts line alone in the updated PR